config-templates/redhat/odf-local/changelog.json

```diff
@@ -3,114 +3,114 @@
         "4.13": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "19 September 2023",
+                "cves": "RHSA-2023:4529,CVE-2023-28484,CVE-2023-29469,RHSA-2023:4523,CVE-2023-27536,CVE-2023-28321,RHSA-2023:4498,CVE-2023-34969,RHSA-2023:4520,CVE-2023-32681,RHSA-2023:4524,CVE-2023-2602,CVE-2023-2603",
+                "golang": "1.20.7",
+                "ubi": "8.8-1037",
+                "new-features": [],
+                "fixes": []
+            },
+            "previous-revision": {
                 "revision": "2",
                 "release-date": "03 August 2023",
                 "cves": "CVE-2023-29403,CVE-2023-29404,CVE-2023-29405,CVE-2023-29402,CVE-2023-29406,CVE-2023-1667,CVE-2023-2283,CVE-2023-26604,CVE-2020-24736",
                 "golang": "1.20.6",
                 "ubi": "8.8-1014",
                 "new-features": [],
                 "fixes": []
-            },
-            "previous-revision": {
-                "revision": "1",
-                "release-date": "24 July 2023",
-                "cves": "",
-                "golang": "1.20.5",
-                "ubi": "8.8-860",
-                "new-features": ["ODF 4.13 initial release"],
-                "fixes": []
             }
         }
     },
     {
         "4.12": {
             "status": "supported",
             "latest-revision": {
+                "revision": "5",
+                "release-date": "19 September 2023",
+                "cves": "RHSA-2023:4529,CVE-2023-28484,CVE-2023-29469,RHSA-2023:4523,CVE-2023-27536,CVE-2023-28321,RHSA-2023:4498,CVE-2023-34969,RHSA-2023:4520,CVE-2023-32681,RHSA-2023:4524,CVE-2023-2602,CVE-2023-2603",
```

Comment:
Please remove all references similar to `RHSA-2023`. We just need to mention CVEs in this list. Same at all other places.